tf_encrypted/convert/convert.py

```diff
@@ -293,3 +303,33 @@ def strip_tensor_info(node_name: str) -> str:
 
 class InvalidArgumentError(Exception):
   pass
+
+
+def find_output_names(pb_trimmed, node_name):
+  """
+  List ouput names for a specific node.
+
+  Add to the output name list if the input name starts with the
+  node name (namespace) of the ops we are registering but
+  not included in the name of the node we are inspecting.
+
+  For example, for the op `split`, based on the node below, the
+  output names are `split` and `split:1`
+
+  node {
+    name: "output/input"
+    op: "Pack"
+    input: "split"
+    input: "split:1"
+    }
+  """
+  output_node = []
+  for n in pb_trimmed:
```

Comment:
wish we could do this without iterating through the entire protobuf. I feel like we should be doing a forward lookahead from the current node that breaks when it stops encountering tensors that are outputs of the current node)

Reply:
this would assume that the nodes are ordered topologically, but if so that would be quite a bit faster

Reply:
agreed, good point.

Reply:
I tweaked the function so it doesn't go through the entire pb file. Let me know what you think. Thanks.